GOVERNANCE.md

```diff
@@ -1,3 +1,175 @@
+* Compiler: [Ben Harshbarger](https://github.com/benharsh)
+* DevOps: [Tim Zinsky](https://github.com/tzinsky)
+* GPU: [Engin Kayraklioglu](https://github.com/e-kayrakli)
+* Performance: [Andy Stone](https://github.com/stonea)
+* Runtime / Platform / Portability: [John Hartman](https://github.com/jhh67)
+* User Support / Language Stabilization: [Lydia Duncan](https://github.com/lydia-duncan)
+
+
+Core Contributors
+-----------------
+
+Role: Core contributors are those who have merge privileges to the
+[main Chapel repository](https://github.com/chapel-lang/chapel).
+These team members are expected to participate in reviewing PRs for
+appropriateness, consistency, and potential security issues.  Core
+contributors are also typically expected to help with triaging
+failures in the nightly regression tests, though exceptions are often
+made for those who are also serving as managers on the team or
+spending most of their time on other projects.
+
+All PRs are expected to be reviewed by at least one core contributor
+other than the PR's author; specifically, PRs created by a core
+contributor should typically be reviewed by another core contributor.
+An exception exists for PRs considered "trivial"—so simple or
+straightforward that asking for a review would feel like a waste of a
+colleague's time rather than a benefit to the project.  When in doubt,
+a review should be requested.  Core contributors who don't feel
+confident reviewing a PR will typically call in additional core
+contributors to help with the review or provide additional thoughts.
+A core contributor who authors or merges a PR is typically responsible
+for any testing fallout caused by that PR, typically by involving the
+PR's author when it was not theirs.  In some cases, the reviewing core
+contributor or another volunteer may step up to help out.
+
+At present, the list of core contributors includes all of the
+technical leaders listed previously, as well as:
+
+* [Jade Abraham](https://github.com/jabraham17)
+* [Soohoon Choi](https://github.com/soohoonchoi)
+* [Matt Drozt](https://github.com/MattToast)
+* [Daniel Fedorin](https://github.com/DanilaFe)
+* [Michael Ferguson](https://github.com/mppf)
+* [Abhishek Girish](https://github.com/agirish)
+* [Shreyas Khandekar](https://github.com/ShreyasKhandekar)
+* [Vass Litvinov](https://github.com/vasslitvinov)
+* [David Longnecker](https://github.com/dlongnecke-cray)
+* [Ben McDonald](https://github.com/bmcdonald3)
+* [Ahmad Rezaii](https://github.com/arezaii)
+* [Anna Rift](https://github.com/riftEmber)
+* [Elliot Ronaghan](https://github.com/ronawho)
+* [Michelle Strout](https://github.com/mstrout)
+* [Karlon West](https://github.com/karlonw)
+
+
+Decision-Making
+---------------
+
+Generally speaking, decision-making on the Chapel project is done by
+the technical leadership and core contributors in this section on a
+consensus basis, taking the opinions of end-users into account when
+the decision is likely to affect them.
+
```

Comment:
Should this section say something about the subteam leads?

Reply:
Maybe:

> Project-wide decisions tend to involve all
technical leads and committers, while those that are specific to a
given effort can be made by a subteam lead and its members, keeping
the tech lead apprised of key decisions.

?  I also added:

> Votes are rarely taken in a binding manner, but are often used to
gauge the level of support for an idea or direction.  In such votes,
those in the minority are typically given the opportunity to continue
to push for alternative approaches, or to concede to the majority,
having registered their opinion.

Reply:
These seem reasonable to me